com.unity.renderstreaming/Documentation~/en/faq.md

```diff
@@ -49,15 +51,15 @@ Currently, version 1.0 does not support simulcast.
 
 ### Can you broadcast the feed from multiple cameras?
 
-Currently, version 1.0 cannot.
+It is available. If you use the hardware encoder, make sure the limitation of the number of cameras using at the same time. See the NVIDIA [document]](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix).
 
 ### How do you increase the video quality?
 
-Video quality depends on the quality of the network. 
+You can set the resolution of the frame on the `CameraStreamer` inspector. Please make sure video quality depends on the quality of the network. 
 
 ### Is Realtime-Raytracing supported? 
 
-Currently, version 1.2 cannot.
+It is available. Please see the detail on the HDRP [document](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@7.1/manual/Ray-Tracing-Getting-Started.html).
```

Comment:
~It is available~

Yes.

Reply:
This one is missing from the previous fix.

Reply:
Fixed it